packages/prisma-client/prisma/schema.prisma

```diff
@@ -15,6 +15,11 @@ model Team {
   users User[]
 }
 
+enum Location {
```

Comment:
can we please not use enums?

just use a `type Location = 'FS' | 'REMOTE'`

Reply:
That doesn't work in Prisma :(

You either do enums or a string

Reply:
you always pass a string in the end to a consumer

Reply:
type is only there to make sure you pass the right thing

Reply:
So you would prefer prisma to have a string and then we just create the location type?